src/test/resources/com/commercetools/sync/products/helpers/variantReferenceResolver/withsetofnestedattributes/withcustomerreferences/with-set-of-references.json

```diff
@@ -0,0 +1,48 @@
+{
+  "key": "key",
+  "sku": "sku",
+  "attributes": [
+    {
+      "name": "attribute-name",
+      "value": [
+        [
+          {
+            "value": [
+              {
+                "id": "key1",
+                "typeId": "customer"
+              },
+              {
+                "id": "key2",
+                "typeId": "customer"
+              }
+            ],
+            "name": "nested-attribute-1-name"
+          },
+          {
+            "value": {
+              "id": "key2",
```

Comment:
key2 is also in nested-attribute-1-name, is it on purpose?

Reply:
yes it is on purpose.